tests/unit/conftest.py

```diff
@@ -76,13 +81,13 @@ async def mattermost_service_mock(
     yield aioresponse_mocker
 
 
-@pytest.fixture()
-async def portainer_service_mock(
+@pytest.fixture
+def portainer_service_mock(
     aioresponse_mocker: aioresponses,
     bearer_code: str,
-    portainer_stacks: Dict[str, Any],
-    valid_config: Dict[str, Any],
-) -> aioresponses:
+    portainer_stacks: dict[str, Any],
+    valid_config: dict[str, Any],
+) -> Iterator[aioresponses]:
```

Comment:
Did you manually change this type or did some tooling do it? Was the original type simply wrong or did something in the logic change with your commit?

Reply:
- the ``Dict`` -> ``dict`` was automatic (pre-commit hook with [pyupgrade](https://github.com/asottile/pyupgrade))
- if you ``yield`` you need to annotate with an ``Iterator`` (these I did as I saw them)